api/app/src/command/medical/patient/delete-patient.ts

```diff
@@ -2,28 +2,60 @@ import cwCommands from "../../../external/commonwell";
 import { validateVersionForUpdate } from "../../../models/_default";
 import { BaseUpdateCmdWithCustomer } from "../base-update-command";
 import { getPatientOrFail } from "./get-patient";
+import { Config } from "../../../shared/config";
+import { makeS3Client } from "../../../external/aws/s3";
+import { Patient } from "../../../models/medical/patient";
+import { getDocuments } from "../../../external/fhir/document/get-documents";
+import { createS3FileName } from "../../../shared/external";
+import { makeFhirApi } from "../../../external/fhir/api/api-factory";
+import { capture } from "../../../shared/notifications";
 
 export type PatientDeleteCmd = BaseUpdateCmdWithCustomer & {
   facilityId: string;
 };
 
+const s3Client = makeS3Client();
+const s3BucketName = Config.getMedicalDocumentsBucketName();
+
 export const deletePatient = async (patientDelete: PatientDeleteCmd): Promise<void> => {
   const { id, cxId, facilityId, eTag } = patientDelete;
 
   const patient = await getPatientOrFail({ id, cxId });
   validateVersionForUpdate(patient, eTag);
 
-  try {
-    // TODO: #393 move to declarative, event-based integration
-    // Synchronous bc it needs to run after the Patient is deleted (it needs patient data from the DB)
-    await cwCommands.patient.remove(patient, facilityId);
+  if (Config.isSandbox()) {
+    const fhirApi = makeFhirApi(cxId);
 
-    //eslint-disable-next-line @typescript-eslint/no-explicit-any
-  } catch (err: any) {
-    // if its a 404, continue without surfacing the error, the patient is not in CW anyway
-    if (err.response?.status !== 404) throw err;
-    console.log(`Patient not found @ CW when deleting ${patient.id} , continuing...`);
+    try {
+      // TODO: #393 move to declarative, event-based integration
+      // Synchronous bc it needs to run after the Patient is deleted (it needs patient data from the DB)
+      await cwCommands.patient.remove(patient, facilityId);
+      await fhirApi.deleteResource("Patient", patient.id);
+      await removeDocsFromS3(patient);
```

Comment:
Now that we're not storing docs in S3 for a specific patient, but are rather using static data, do we still need this?